tests/sample_process.py

```diff
@@ -17,6 +23,12 @@
         sys.stdout.write("out ouch")
         sys.stderr.write("err output on failure")
         sys.exit(11)
+    # check env
+    if len(sys.argv) == 3 and sys.argv[1]=='check' and sys.argv[2]=='env':
+        if os.environ.get('GELKIPWDUZLOVSXE') == '1':
```

Comment:
Just a random string really. Although I'm sure we can come up with a backronym 😁